library/Notifications/Web/Control/SearchBar/ObjectSuggestions.php

```diff
@@ -199,9 +199,10 @@ protected function fetchColumnSuggestions($searchTerm)
                 }
 
                 $relation = $isIdTag ? 'object.tag' : 'object.extra_tag';
+                /** @var string $tagName */
```

Comment:
Please take the chance to document this already in the model. Saves us a few more inline docs in the future.